### PR TITLE
Add Trapeze To Shapes

### PR DESCRIPTION
**Pull request by usernamemyname**
### Description
I add new trapeze shape. It can help to make some auras with comfort. 
File type of this shape is TGA as recommended.

### CODE REVIEW

**Comment by InfusOnWoW** [comment]
Trapeze

**Comment by InfusOnWoW** [comment]
Well, you can't see it but if you click on it that's the added image. The gray parts are actually transparent.

**Comment by InfusOnWoW** [comment]
I'm not too sure if we want to have that as a default texture. Can you show us a few screenshots in action?

File: WeakAuras/Types.lua [hunk 1]
```diff
@@ -956,6 +956,7 @@ WeakAuras.texture_types = {
     ["Interface\\AddOns\\WeakAuras\\Media\\Textures\\Square_White_Border"] = "Square with Border",
     ["Interface\\AddOns\\WeakAuras\\Media\\Textures\\Square_FullWhite"] = "Full White Square",
     ["Interface\\AddOns\\WeakAuras\\Media\\Textures\\Triangle45"] = "45° Triangle",
+    ["Interface\\AddOns\\WeakAuras\\Media\\Textures\\Trapeze"] = "Trapeze",
```
Comment (emptyrivers) [inline WeakAuras/Types.lua]: Trapezoid, not trapeze, please.

**Comment by emptyrivers** [comment]
Yeah, I agree. Any user can add this texture in themselves without too much trouble. I’d like to see an example aura which really benefits from this kind of design before including a trapezoid.

**Comment by usernamemyname** [comment]
Dark Inquisitor Xanesh

**Comment by usernamemyname** [comment]
Overlay for mechanics with a sphere on the Dark Inquisitor Xanesh

**Comment by Stanzilla** [comment]
I like it! Thank you for that!

**Pull request merged**